src/mds/topology/topology_metric.h

```diff
@@ -186,6 +188,8 @@ struct LogicalPoolMetric {
     bvar::Status<uint64_t> writeIOPS;
 
     explicit LogicalPoolMetric(const std::string &logicalPoolName) :
+        serverNum(kTopologyLogicalPoolMetricPrefix,
+            logicalPoolName + "_server_num", 0),
         chunkServerNum(kTopologyLogicalPoolMetricPrefix,
             logicalPoolName + "_chunkserver_num", 0),
         copysetNum(kTopologyLogicalPoolMetricPrefix,
```

Comment:
Why not remove the extra `_ `here?

Reply:
> Why not remove the extra `_ `here?

The cluster metric prefix is 'topology_metric_cluster_' which end with '', but pool metric is pool prefix 'topology_metric_logicalPool' + poolName which not end with '-'.